lib/unit/models/authorization_request/decline_request.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# class for declining the authorization request
+# @see https://docs.unit.co/cards-authorization-requests/#decline-authorization-request
+module Unit
+  module AuthorizationRequest
+    class DeclineRequest
+      attr_reader :authorization_request_id, :reason
+
+      # @param authorization_request_id [String]
+      # @param reason [String]
+      def initialize(authorization_request_id, reason)
+        @authorization_request_id = authorization_request_id
+        @reason = reason
```

Comment:
@AsafPolakovski do we want to add something like an enum for reason? as it can only be one of a list of strings